services/virtual-assistant/src/virtual_assistant/assistant/watson.py

```diff
@@ -156,6 +178,15 @@ async def send_message(self, message: AssistantInput) -> AssistantOutput:
             session_id=message.session_id,
             user_id=message.user_id,
             input=message_input,
+            context=MessageContext(
+                skills=MessageContextSkills(
+                    actions_skill=MessageContextActionSkill(
+                        skill_variables={
+                            _WATSON_DRAFT_ENVIRONMENT_VARIABLE: self.variables.draft
+                        }
```

Comment:
@josejulio I think we are setting Draft to always be true, is that right? Or, did I miss something? 🤔

Reply:
Thanks! I was a little confused where the injection was actually happening.